packages/core/src/ceramic.ts

```diff
@@ -630,11 +631,14 @@ class Ceramic implements CeramicApi {
     const doc = await this._repository.load(docRef.baseID, opts)
 
     // If DocID is requested, return the document
-    if (docRef instanceof DocID) {
-      return doc
-    } else {
+    if (docRef instanceof CommitID) {
       // Here CommitID is requested, let's return document at specific commit
       return doc.rewind(docRef)
+    } else if (opts.atTime) {
+      const commitId = doc.findCommitAt(opts.atTime)
```

Comment:
This, with `findCommitAt` do not belong here IMO, but let it live for now. Going to move it out of the document after the merge.

Reply:
Ok, let me know where I should move it? Or do you want to later?

Reply:
We'll do it later. It's all about having snapshots, which seems to be pretty targeted change. For now let it live here.